rollup.config.js

```diff
@@ -95,4 +95,11 @@ export default [
   createCommonJSConfig('src/shallow.ts', 'dist/shallow.js'),
   createCommonJSConfig('src/middleware.ts', 'dist/middleware.js'),
   createCommonJSConfig('src/vanilla.ts', 'dist/vanilla.js'),
+
+  // Create a purely ESM version, can be imported from `zustand/esm`
+  createESMConfig('src/vanilla.ts', 'dist/vanilla.module.js'),
+  createESMConfig('src/shallow.ts', 'dist/shallow.module.js'),
+  createESMConfig('src/index.ts', 'dist/index.module.js'),
```

Comment:
```suggestion
```

oops, we don't need this. it's `index.js`. this is inverted for historical reason.